C/c4Database.cc

```diff
@@ -324,8 +316,8 @@ Retained<C4Document> C4Database::getDocumentBySequence(C4SequenceNumber sequence
 }
 
 Retained<C4Document> C4Database::putDocument(const C4DocPutRequest &rq,
-                                 size_t* C4NULLABLE outCommonAncestorIndex,
-                                 C4Error *outError) {
+                                             size_t *C4NULLABLE outCommonAncestorIndex,
```

Comment:
This spacing is weird because the `*` has nothing to do with the C4NULLABLE keyword, it's part of the `size_t`.

Reply:
This one is difficult. We'll have to discuss more. I don't think Clang-Format has an option to cover this case.